generator/lookml.py

```diff
@@ -129,42 +152,98 @@ def _lookml(namespaces, glean_apps, target_dir, dryrun, namespace_filter=[]):
     with open(target / "namespaces.yaml", "w") as target_namespaces_file:
         target_namespaces_file.write(namespaces_content)
 
+    run_generation = partial(_run_generation, metric_hub_repos)
+
+    generate_views = []
+    generate_datagroups = []
     v1_mapping = _glean_apps_to_v1_map(glean_apps)
+    logging.info("  Generating views and datagroups")
     for namespace, lookml_objects in _namespaces.items():
         if len(namespace_filter) == 0 or namespace in namespace_filter:
-            logging.info(f"\nGenerating namespace {namespace}")
-
             view_dir = target / namespace / "views"
             view_dir.mkdir(parents=True, exist_ok=True)
             views = list(
                 _get_views_from_dict(lookml_objects.get("views", {}), namespace)
             )
 
-            logging.info("  Generating views")
             v1_name: Optional[str] = v1_mapping.get(namespace)
-            for view_path in _generate_views(view_dir, views, v1_name, dryrun=dryrun):
-                logging.info(f"    ...Generating {view_path}")
+            for view in views:
+                generate_views.append(
+                    partial(
+                        _generate_view,
+                        view_dir,
+                        view,
+                        v1_name,
+                        dryrun,
+                    )
+                )
+                generate_datagroups.append(
+                    partial(
+                        generate_datagroup,
+                        view,
+                        target,
+                        namespace,
+                        dryrun,
+                    )
+                )
 
-            logging.info("  Generating datagroups")
-            generate_datagroups(views, target, namespace, dryrun=dryrun)
+    with Pool(parallelism) as pool:
```

Comment:
I tested this with `ThreadPool` as well, but using `Pool` it was about 2x faster when I tested it locally